src/components/webview/SourceControlPanelProvider.ts

```diff
@@ -175,6 +179,29 @@ export class SourceControlPanelProvider {
 
 						this.__webviewPanel.reveal(undefined, preserveFocus);
 					}
+
+					if (message.kind === 'webview.sourceControl.createIssue') {
+						const storedClerkToken =
+							this.__userService.getLinkedToken();
+
+						if (storedClerkToken === null) {
```

Comment:
you can return here if you invert the condition

Reply:
i refactored this block